publish.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Yeah, we probably won't hit the inflection point of needing something like that for a while.

Changes LGTM